lib/supavisor.ex

```diff
@@ -349,4 +349,36 @@ defmodule Supavisor do
       pid -> Manager.set_parameter_status(pid, ps)
     end
   end
+
+  @spec get_pool_ranch(id) :: {:ok, map()} | {:error, :not_found}
+  def get_pool_ranch(id) do
+    case :syn.lookup(:tenants, id) do
+      {_sup_pid, %{port: _port, host: _host} = meta} -> {:ok, meta}
+      _ -> {:error, :not_found}
+    end
+  end
+
+  @spec start_local_server(map()) :: {:ok, map()} | {:error, any()}
+  def start_local_server(%{max_clients: max_clients} = args) do
+    # max_clients=-1 is used for testing the maximum allowed clients in ProxyTest
+    {acceptors, max_clients} =
+      if max_clients > 0,
+        do: {round(max_clients / 100), max_clients},
```

Comment:
What if `max_clients` is smaller than `50`? We will end with no `acceptors`?

Reply:
right, must be `ceil`